src/actions/arcActions.ts

```diff
@@ -301,7 +297,7 @@ async function getProposalDetails(daoInstance: Arc.DAO, votingMachineInstance: A
     boostedVotePeriodLimit: Number(proposalDetails.currentBoostedVotePeriodLimit),
     contributionDescriptionHash: contributionProposal.contributionDescriptionHash,
     daoAvatarAddress: avatarAddress,
-    description,
+    description: url,
```

Comment:
hmm, was there a final discussion around whether we will always be storing a URL in IPFS. Seems like one the main advantages of IPFS is that we could store the full description there and make sure it cant be changed after the fact

Reply:
I absolutely want to keep the option to attach URLs to proposals, the commenting feature of google docs & flexibility of pointing to other platforms is too useful to do away with.

But I agree, an additional "description" field on the IPFS object would be good to finalize some chunk of text as a description & including both fields would give us best of both. Maybe we could throw a proposal's abstract/summary into it's description on IPFS and have a longer-form description with comments attached to a google doc via the url.

Reply:
there's this doc which can also be the locus for this discussion: https://docs.google.com/document/d/1mnhU_L5opxMGuCcdtgOb3D40KX4jquROnx2HvP1qG50/edit#